eng/release/DiagnosticsReleaseTool/DiagnosticsReleaseCommandLine.cs

```diff
@@ -73,12 +82,32 @@ static async Task<int> Main(string[] args)
                     Path.Join(Path.GetTempPath(), Path.GetRandomFileName())))
             .LegalFilePathsOnly();
 
-        private static Option<string> PublishPathOption() =>
+        private static Option<string> AzureStorageAccountNameOption() =>
             new Option<string>(
-                aliases: new[] { "-o", "--publish-path" },
-                description: "Path to publish the generated layout and publishing manifest to.")
+                aliases: new[] { "-n", "--account-name" },
+                description: "Storage account name, must be in public azure cloud.")
             {
-                IsRequired = true
+                IsRequired = true, 
             };
+
+        private static Option<string> AzureStorageAccountKeyOption() =>
+            new Option<string>(
+                aliases: new[] { "-k", "--account-key" },
+                description: "Storage account key, in base 64 format.")
+            {
+                IsRequired = true,
+            };
+
+        private static Option<string> AzureStorageContainerNameOption() =>
+            new Option<string>(
+                aliases: new[] { "-c", "--container-name" },
+                description: "Storage account container name where the files will be uploaded.",
+                getDefaultValue: () => "dotnet-monitor");
+
+        private static Option<int> AzureStorageSasExpirationOption() =>
+            new Option<int>(
+                aliases: new[] { "--sas-valid-days" },
+                description: "Number of days to allow access to the blobs via the provided SAS URIs.",
+                getDefaultValue: () => 732); // default to just over 2 years
```

Comment:
I haven't worked with sas tokens that are based on policies, but normal sas tokens are tied to one of the account keys. Regularly scheduled key rotations would invalidate all the sas tokens (which normally isn't an issue since the tokens last much much less than 2 years). Would such rotation not affect these sas tokens?

Reply:
That would affect these keys as well, the idea is that you don't need to do a key rotation at all because you can edit the policy to revoke access to an individual asset.